apps/playlist/pkg/app/app.go

```diff
@@ -76,12 +76,12 @@ func New(cfg app.Config) (app.App, error) {
 	return a, nil
 }
 
-func GetKinds() map[schema.GroupVersion]resource.Kind {
+func GetKinds() map[schema.GroupVersion][]resource.Kind {
 	gv := schema.GroupVersion{
 		Group:   playlistv0alpha1.PlaylistKind().Group(),
 		Version: playlistv0alpha1.PlaylistKind().Version(),
```

Comment:
Would it be possible to generate constants for this that is shared. This part now looks a bit strange if you have several resources under the same `GroupVersion`.

Reply:
i created an issue for this: https://github.com/grafana/grafana-app-sdk/issues/515